open_rarity/rarity_ranker.py

```diff
@@ -20,35 +27,75 @@ def rank_tokens(*, token_id_to_scores: dict[str, float]) -> dict[str, int]:
 
         Parameters
         ----------
-        token_ids_to_score : dict[str, float]
-            A dictionary of token ids (this can be whatever id you want, as long
-            as its unique) to the rarity score.
+        collection : Collection
+            Collection object with populated tokens
+        scorer: Scorer
+            Scorer instance
 
         Returns
         -------
-        dict[str, int]
-            A dictionary of token ids to their ranks. More specificially, it is the
-            provided tokens ids to the rank it has within the set of possible
-            tokens defined by @token_id_to_scores.
+        list[TokenRarity]
+            list of TokenRarity objects with score, rank and token information
```

Comment:
Right now, this implementation is returning a list of token rarity sorted by "rank" - let's put that in the description of the function